client/src/components/DataFiles/DataFilesModals/DataFilesPreviewModal.js

```diff
@@ -1,11 +1,11 @@
-import React, { useState, useCallback, useEffect } from 'react';
+import React, { useState, useCallback } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { Modal, ModalHeader, ModalBody } from 'reactstrap';
-import { LoadingSpinner } from '_common';
+import { LoadingSpinner, Message } from '_common';
 import './DataFilesPreviewModal.module.scss';
 
 const PreviewModalSpinner = () => (
-  <div className="h-100 listing-placeholder">
+  <div styleName="loading-container">
     <LoadingSpinner />
   </div>
 );
```

Comment:
I think we can lose this function and wrapper around `<LoadingSpinner>`, and directly insert it into the `<ModalBody>` below. Currently I am seeing the spinner now vertically centered, but if you remove the wrapper it is vertically centered.

Reply:
good catch.  fixed!